scintillometry/utils/phase_utils.py

```diff
@@ -0,0 +1,142 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import warnings
+
+import numpy as np
+import astropy.units as u
+import astropy.time as time
+
+try:
+    from .pint_utils import PintToas, make_toa_list
```

Comment:
I think these import should be in `PintPhase__init__` - this file should not bypass the import failures.

Reply:
importing them in the `PintPhase.__init__` does not keep the modules after the` __init__`. In` _make_toas`, I will need to use make_toa_list. So moving `import make_toa_list` to ` _make_toas()`